FlowCrypt/Functionality/Services/Remote Pub Key Services/PubLookup.swift

```diff
@@ -8,11 +8,13 @@
 
 protocol PubLookupType {
     func lookup(email: String) async throws -> RecipientWithSortedPubKeys
+    func fetchRemoteUpdateLocal(with email: String) async throws -> RecipientWithSortedPubKeys
 }
 
 class PubLookup: PubLookupType {
     private let wkd: WkdApiType
     private let attesterApi: AttesterApiType
+    private let localContactsProvider: LocalContactsProviderType?
```

Comment:
Would it be a large change to always require it? It would be more consistent with the rest of the app - I think it would be worthwhile to do.